admin/js/google-fonts.js

```diff
@@ -0,0 +1,138 @@
+const API_URL = "https://www.googleapis.com/webfonts/v1/webfonts?key=";
+const API_KEY = "AIzaSyC9-sErFQouy_mc0OqUmkNHEJZiZ-2_3iI"; // The API key is embed here because the API doesn't seem to have limits or costs.
```

Comment:
Who's API key is this?  

While there might not be limits is this appropriate?  It would be unfortunate to require a user to have to enter this to use the utility but I'm not sure what the rules around distributing the key are.

Can we maybe add a place for a user to paste their own key (and maybe a link/simple instructions to obtain it) instead?  And (if I understand) that key is only needed to get a current list of typefaces... without the key this would still work with the backup JSON right?

Reply:
The API url was changed to use a proxy to avoid sharing the API key.

Reply:
I do not think it is best to use vercel as proxy, because of the bandwidth limits that vercel has. It starts failing once the limit reached.

Also Chrome flags it as Phishing site https://google-fonts-service-proxy.vercel.app
<img width="494" alt="image" src="https://user-images.githubusercontent.com/1935113/193239624-0ca061a7-5702-4a2a-acf9-6378362e4c2e.png">

This could be fixed by updating the site, but for the above reason it should be avoided.

Reply:
Also, this is adding another layer of dependency on a single user. 
API key of the user + vercel account of the user
For some reason, if vercel decides to suspend user account, this feature fails.

**Update:** I looked into the vercel fair usage policy guidelines. It suggests to not use their service as proxy.
https://vercel.com/docs/concepts/limits/fair-use-policy

Reply:
It seems unlikely to reach the 100GB traffic quota of the free account. Apart from that, I think this can be considered " Functions that query DBs or APIs" more than a proxy that would be a general purpose proxy. 

> For some reason, if vercel decides to suspend user account, this feature fails.
The feature won't fail because if there is no answer from the API a static fallback JSON included in the plugin is used.

Reply:
This aspect is not vital for the plugin to work so we can discuss it in another PR. I removed the API fetching code to use just the local JSON file with the Google fonts data until we land a better solution.